setup.cfg

```diff
@@ -40,10 +40,6 @@ install_requires =
     asgiref
     typing_extensions; python_version < "3.8"
     markupsafe>=1.1.1
-    # werkzeug 2.2.0 breaks flask-login. see https://github.com/maxcountryman/flask-login/issues/686 for details.
-    # It causes ImportError: cannot import name 'parse_rule' from 'werkzeug.routing'
-    # we need werkzeug<2.2 limitation until flask_login can handle it
-    werkzeug>=2.0,<2.2
```

Comment:
Since it is a minor release, I would say create a separate PR to just remove `werkzeug` dependency and merge it to main and we can cherry pick it to our 1.8 release branch and release 1.8.1 from there

Reply:
We can then have a separate PR to release 1.8.1

Reply:
I created https://github.com/astronomer/astronomer-providers/pull/615